receiver/solacereceiver/model/v1/receive_v1.pb.go

```diff
@@ -223,13 +223,12 @@ func (SpanData_TransactionEvent_Initiator) EnumDescriptor() ([]byte, []int) {
 // nanoseconds, since midnight, Jan. 1, 1970 UTC.
 //
 // Notes on the field numbers used:
-// - Field numbers 1-15 are used for attributes that are expected to be present
```

Comment:
ditto, is this intentional?

Reply:
I ran `make golint` locally to make sure it will pass in CI. This command fails locally with error (it fails locally now on main branch as well):
```
ERRO Running error: 1 error occurred:
	* can't run linter goanalysis_metalinter: gofmt: can't extract issues from gofmt diff output "--- /Users/mar4uk/go/mar4uk/opentelemetry-collector-contrib/receiver/solacereceiver/model/v1/receive_v1.pb.go.orig\t2022-11-14 08:46:32\n+++ /Users/mar4uk/go/mar4uk/opentelemetry-collector-contrib/receiver/solacereceiver/model/v1/receive_v1.pb.go\t2022-11-14 08:46:32\n@@ -223,13 +223,12 @@\n // nanoseconds, since midnight, Jan. 1, 1970 UTC.\n //\n // Notes on the field numbers used:\n-// - Field numbers 1-15 are used for attributes that are expected to be present\n-//   on the wire with every single message not containing an error_description.\n-//   Special priority is given to fields that can be repeated.\n-// - Field numbers 16+ are used for other attributes.\n+//   - Field numbers 1-15 are used for attributes that are expected to be present\n+//     on the wire with every single message not containing an error_description.\n+//     Special priority is given to fields that can be repeated.\n+//   - Field numbers 16+ are used for other attributes.\n //\n // Next available field ID: 39\n-//\n type SpanData struct {\n \tstate         protoimpl.MessageState\n \tsizeCache     protoimpl.SizeCache\n": can't parse patch: parsing time "2022-11-14 08:46:32" as "2006-01-02 15:04:05 -0700": cannot parse "" as "-0700"
```
That's why I ran `make gofmt` locally and commited changes. Interesting that this error doesn't occur in CI...

Reply:
reverted